### DISCUSSION TITLE
Better Editor for VPY

### DISCUSSION THREAD

**Discussion by fmohican** [staxrip/staxrip#419] [Ideas]
Its possible to have better 'editor' (GUI) for avs/vpy code?
Like https://github.com/mysteryx93/VapourSynthViewer.NET ?

**Comment by stax76** [comment]
What you can try is the _Join Filters_ feature of the code editor.

**Comment by JJKylee** [comment]
I do miss a VapourSynth counterpart for [AvsPmod](https://github.com/gispos/AvsPmod) in AviSynth.

BTW, [VapourSynth Editor](https://bitbucket.org/mystery_keeper/vapoursynth-editor/) does a decent job.
Simple but useful. It provides preview as well.
